bear-requirements.txt

```diff
@@ -11,6 +11,7 @@ cpplint~=1.3
 dennis~=0.9
 docutils-ast-writer~=0.1.2
 eradicate~=0.1.6
+fake-useragent~=0.1.11
```

Comment:
`headers` present in `fake-useragent` are pretty old. Do we really want to use it?